### ISSUE TITLE
[Button]: interactive parts in the button are not clickable

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1544]
### Description
If the [Button] OR [Link Button] consists of parts, the focus is not visible when navigating from the keyboard. The system recognizes the button as a whole

### Precondition
Screen Reader is **OFF** or **ON**

### Steps to Reproduce
1. Open [Button](https://uui.epam.com/documents?category=components&id=button&mode=propsEditor&skin=UUI3_loveship) OR [Link Button](https://uui.epam.com/documents?category=components&id=linkButton&mode=propsEditor&skin=UUI3_loveship) component.
2. Set property `onClear = callback`.
3. Select any icon in the property with the same name.
4. Set property `onIconClick = callback`.
5. Move to the button using "Tab" key.
6. Press "Enter" when the focus is on the button.

### Actual result
System recognizes the button with parts as a whole if User uses the keyboard.

https://user-images.githubusercontent.com/57751283/204541494-67dd0632-22af-49ee-ac3b-4f24374a9722.mp4

https://github.com/epam/UUI/assets/57751283/07e4f874-c5bb-4f7f-88c3-2f3b5bd37b90

### Expected result
System recognizes the parts of the button and highlights the focus for each part separately.

**Failed WCAG Checkpoint:** 2.1.1 Keyboard (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

**Comment by NatalliaAlieva** [comment]
As a button itself must have any action, it means that the button can't have any internal action. According to a11y there can't be a button inside a button. This case is valid only for badges or tags, where the badge itself is not clickable, but the icon inside it is clickable. Therefore, we will not do any support for such cases. If projects need accessible buttons, please do not use this variant.
